src/mainPageTB.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import InboxIcon from '@material-ui/icons/Inbox';
+import DraftsIcon from '@material-ui/icons/Drafts';
+
+
+import './mainPageTB.css';
+import Button from '@material-ui/core/Button';
+const styles = theme => ({
+    root: {
+        width: '100%',
+        maxWidth: 360,
+        backgroundColor: theme.palette.background.paper,
+    },
+});
+
+function MainPageTB(props) {
+    const { classes } = props;
+
+    let contrib = props.contributions;
+    console.log(Object.keys(contrib));
+    return (
+        <div className={classes.root+" MainPage-format"}>
+            <Button variant="outlined" color={"primary"} className={classes.button}>Add Contribution </Button>
+            <List component="nav">
+                {/*Object.keys() gives you the keys of that object as an array*/}
```

Comment:
put comments above relevant lines in JS instead of nested in JSX. remove personal notes